datahub-web-react/src/app/ingest/source/builder/RecipeForm/dbt_cloud.tsx

```diff
@@ -0,0 +1,296 @@
+import React from 'react';
+import styled from 'styled-components';
+import { get } from 'lodash';
+import { RecipeField, FieldType, setFieldValueOnRecipe } from './common';
+
+const TipSection = styled.div`
+    margin-bottom: 12px;
+`;
+
+export const DBT_CLOUD = 'dbt-cloud';
+
+export const DBT_CLOUD_TOKEN: RecipeField = {
+    name: 'token',
+    label: 'API Token',
+    tooltip:
+        'A service account API token for extracting metadata from dbt Cloud APIs. This token must have the privileges required to read metadata (Metadata permission set).',
+    type: FieldType.SECRET,
+    fieldPath: 'source.config.token',
+    placeholder: 'dbts_ndg_m5oCuSRRC80tpx4ysYfN2tOreiHuATAu5VFcdrkIznQgl4VCOs6w==',
+    required: true,
+    rules: null,
+};
+
+export const DBT_CLOUD_ACCOUNT_ID: RecipeField = {
+    name: 'account_id',
+    label: 'Account ID',
+    tooltip: (
+        <span>
+            <TipSection>The ID of the dbt Cloud account to extract metadata for.</TipSection>
+            <TipSection>
+                This can be found in the URL of your dbt instance: https://cloud.getdbt.com/#/accounts/ACCOUNT_ID/.
+            </TipSection>
+        </span>
+    ),
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.account_id',
+    placeholder: '123',
+    required: true,
+    rules: null,
+};
+
+export const DBT_CLOUD_PROJECT_ID: RecipeField = {
+    name: 'project_id',
+    label: 'Project ID',
+    tooltip: (
+        <span>
+            <TipSection>The ID of the dbt Cloud project to extract metadata for.</TipSection>
+            <TipSection>
+                This can be found in the URL of your dbt instance:
+                https://cloud.getdbt.com/#/accounts/123/projects/PROJECT_ID.
```

Comment:
Should we stick to our previous pattern and put `ACCOUNT_ID` here instead of 123?

Reply:
Yeah I think this may make sense